staging/src/k8s.io/apiserver/pkg/endpoints/installer.go

```diff
@@ -374,10 +374,7 @@ func (a *APIInstaller) registerResourceHandlers(path string, storage rest.Storag
 		shortNames = shortNamesProvider.ShortNames()
 	}
 
-	tableProvider, ok := storage.(rest.TableConvertor)
-	if !ok {
-		tableProvider = rest.DefaultTableConvertor
```

Comment:
Add a panic or something here so that people who aren't using the generic store will get at least *some* notification that things won't work right.